README.md

```diff
@@ -33,3 +33,9 @@ npm run dev
 npm run build
 npm run start
 ```
+
+### Docker
+```bash
+sudo docker build -t txtdot/txtdot:1.2.0 .
```

Comment:
**Why are you specifying version again?**  
Just `txtdot/txtdot:latest`. The same for the next line, but "latest" there is extra.